migrations/1085649617_create_category_table.up.sql

```diff
@@ -0,0 +1,34 @@
+-- CREATE TABLE category (
+--     id         int,
+--     name       varchar(100) NOT NULL UNIQUE,
+--     is_delete  BOOLEAN,
+--     created_at DATETIME  DEFAULT CURRENT_TIMESTAMP,
+--     delete_at  TIMESTAMP,
+--     updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+-- );
+
+-- CREATE TABLE category (
+--                        user_id int NOT NULL AUTO_INCREMENT,
+--                        name varchar(255) NOT NULL,
+--                        fullname varchar(255),
+--                        balance int,
+--                        PRIMARY KEY (user_id)
+-- );
+
+CREATE TABLE IF NOT EXISTS category (
+    id INT NOT NULL,
+    name varchar(100) NOT NULL,
+    is_delete  BOOLEAN,
+    created_at TIMESTAMP  DEFAULT CURRENT_TIMESTAMP,
+    delete_at  TIMESTAMP,
```

Comment:
deleteD_at

Reply:
WHERE deleted_at is null
WHERE is_delete = false

is_delete = true AND deleted_at IS NULL;